fsharp-backend/src/LibExecution/StdLib/LibString.fs

```diff
@@ -150,7 +150,7 @@ let fns : List<BuiltInFn> =
         (function
         | _, [ DStr s ] ->
             (try
-              bigint (s |> int) |> DInt |> Value
+              s |> System.Numerics.BigInteger.Parse |> DInt |> Value
```

Comment:
Here you can use `System.Numerics.BigInteger.TryParse` instead of doing a try-catch over the "unsafe" `Parse` function.

You can do:

```
match System.Numerics.BigInteger.TryParse with
| true , v -> v |> DInt |> Value
| false, _ -> Value(errStr ("Expected a string with only numbers")))
```

Alternatively, you can use something like FSharpPlus generic `tryParse` and map it with `Option.map` or stuff like that, if you want to use the combinators approach instead.